addons/payment/models/account_payment.py

```diff
@@ -137,3 +182,22 @@ def action_post(self):
         payments_tx_not_done.action_cancel()
 
         return res
+
+    def action_view_refunds(self):
+        return {
```

Comment:
Sorry, I wasn't clear in my comment. I meant: "return the form view rather than list+form *when* there is only one record to browse, list+form when there are several records".